store/v2/root/snapshot_test.go

```diff
@@ -191,11 +194,19 @@ func TestMultistoreSnapshotRestore(t *testing.T) {
 		require.Equal(t, sourceSubStore, targetSubStore)
 	}
 
-	// restore checking for non-empty store
-	target2 := newMultiStoreWithBasicData(t, memdb.NewDB(), 0)
+	// checking snapshot restore for non-empty store with existed saved version
```

Comment:
Last nit - I think we should leave this as 0 so we test with existing versions separately from an existing schema

Also, should be "existing" rather than existed.

Other than that, LGTM

Reply:
Okay